index.html

```diff
@@ -1,224 +1,305 @@
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-four remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-five remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-six remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
-            <div class="img-seven remaining-img">
-                <h2 class="card-heading-white">Profesional Art Printing Data</h2>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
-                    industry's standard.
-                </p>
-                <ul class="skills-details">
-                    <li>HTML</li>
-                    <li>Bootstrap</li>
-                    <li>Ruby</li>
-                </ul>
-                <button class="project-details" type="button">See project</button>
-            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="card-section">
```

Comment:
- [ ] As requested by the previous reviewer, please kindly implement the dark overlay on the card and when you hover the display changes with the `button` tag on the card. 

- [ ] This is the required [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=HhgzrHa7uTvTkdte-0) when the card has not hovered. 

![Screenshot (189)](https://user-images.githubusercontent.com/78104544/229261094-a7da608b-1c58-4aa0-9894-b1d34e0a58ee.png)

- [ ] **_The design when the card has hovered._**


![Screenshot (190)](https://user-images.githubusercontent.com/78104544/229261188-5b140b3e-20ca-4177-ae92-24e2da5b2055.png)